alpaca_trade_api/rest.py

```diff
@@ -360,23 +365,27 @@ def replace_order(
             stop_price: str = None,
             trail: str = None,
             time_in_force: str = None,
-            client_order_id: str = None
+            client_order_id: str = None,
+            notional: str = None,
     ) -> Order:
         """
         :param order_id:
-        :param qty: str of int
+        :param qty: str of float
         :param limit_price: str of float
         :param stop_price: str of float
         :param trail: str of float, represents trailing_price or
                trailing_percent. determined by the original order.
         :param time_in_force: day, gtc, opg, cls, ioc, fok
+        :param notional: str of float
 
         note: you cannot replace type of order. so, it was trailing_stop(e.g)
               it will remain trailing_stop.
         """
         params = {}
         if qty is not None:
             params['qty'] = qty
+        if notional is not None:
+            params['notional'] = notional
```

Comment:
It doesn't seem that the Orders API supports replacement with fractional quantities or notional amounts (see [gobroker/oms/orderapi.go#L210-213](https://github.com/alpacahq/gobroker/blob/765a9fdbc1c81fd7abe79b7c317afff8c2c75a45/oms/orderapi.go#L210)), so I'm gonna revert this.